docs/install-on-gke.md

```diff
@@ -0,0 +1,33 @@
+# Install on GKE
+
+## Pre-requisites
```

Comment:
Add a note which says you will need a docker CLI master for now, and in the future at least 19.03. We need this PR https://github.com/docker/cli/pull/1583 which has been merged after the 18.09 release.

Reply:
+1

Reply:
We will replace the note with "please use docker CLI 19.03+" when it is out.

Reply:
Done